src/ui/buttons/script.jsx

```diff
@@ -145,6 +144,46 @@ function getComponentScript() : () => void {
             }
         }
 
+        function addDivideLogoAnimationExperiment() {
+            const buttonElement = document ? document.querySelector('.paypal-button-label-container') : null;
```

Comment:
Instead of using a Ternary could you do this instead? Or is it important that the default value is null?
`const buttonElement = document && document.querySelector('.paypal-button-label-container');`

Reply:
it does not matter if it is null or a boolean

Reply:
Using && is easier to read than a ternary so we generally avoid ternaries unless they are necessary